src/Upgrade/Infrastructure/Dto/SourceCodeProvider/PullRequestDto.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrade\Infrastructure\Dto\SourceCodeProvider;
+
+class PullRequestDto
+{
+    /**
+     * @var string
+     */
+    protected string $sourceBranch;
+
+    /**
+     * @var string
+     */
+    protected string $targetBranch;
+
+    /**
+     * @var string
+     */
+    protected string $title;
+
+    /**
+     * @var ?string
+     */
+    protected ?string $body;
+
+    /**
+     * @var bool
+     */
+    protected bool $autoMerge = false;
+
+    /**
+     * @param string $sourceBranch
+     * @param string $targetBranch
+     * @param string $title
+     */
+    public function __construct(string $sourceBranch, string $targetBranch, string $title)
```

Comment:
Please add `autoMerge` property as the last constructor argument with default value.

Reply:
++